htdocs/modules/system/icms_version.php

```diff
@@ -66,7 +66,7 @@
 /**  Development information */
 	'status_version' => "RC",
```

Comment:
Should probably change this to beta, too

Reply:
As a matter of fact, this would be RC2, instead of Beta 2. We're coming from RC1.